sycl/test-e2e/Matrix/joint_matrix_down_convert_impl.hpp

```diff
@@ -0,0 +1,165 @@
+#define TM 8
+#define TN SG_SZ
+#define TK 16
+
+#define BF16_EPSILON 0.00781250
+
+template <typename T, size_t NUM_ROWS, size_t NUM_COLS> struct big_matrix {
```

Comment:
Would be really good to move this common class out of tests into something like helper.hpp to stop duplicating it.